user_guide_src/source/outgoing/view_cells.rst

```diff
@@ -154,7 +161,7 @@ If you need more control over the rendering of the HTML, you can implement a ``r
 Computed Properties
 ===================
 
-If you need to perform additional logic for one or more properties you can use computed properties. These require setting the property to either ``protected`` or ``private`` and implementing a public method whose name consists of the property name surrounded by ``get`` and ``Property``.
+If you need to perform additional logic for one or more properties you can use computed properties. These require setting the property to ``protected`` and implementing a public method whose name consists of the property name surrounded by ``get`` and ``Property``.
```

Comment:
Why `private` property was removed? It should still be a valid option.

Reply:
Because it does not work.
```
Cannot access private property App\Cells\AlertMessageCell::$type
in SYSTEMPATH/Traits/PropertiesTrait.php on line 35.
 1 SYSTEMPATH/View/Cell.php(214): CodeIgniter\View\Cells\Cell->fill([...])
 2 SYSTEMPATH/View/Cell.php(99): CodeIgniter\View\Cell->renderCell(Object(App\Cells\AlertMessageCell), 'render', [...])
 3 SYSTEMPATH/Common.php(1200): CodeIgniter\View\Cell->render('AlertMessageCell', [...], 0, 'AppCellsAlertMessageCellrender3ab314af93d2044b8409e6348dd9ff46')
```
Ah, is this a bug?

Reply:
Hmm... it worked on CI 4.3.0. I guess we have some regression. I'll try to check later if this was an intentional change, and we just didn't include it in the user guide or a bug, though.

Reply:
The following test fails in both `develop` and `v4.3.0`.
```diff
--- a/tests/_support/View/Cells/ListerCell.php
+++ b/tests/_support/View/Cells/ListerCell.php
@@ -15,7 +15,7 @@ use CodeIgniter\View\Cells\Cell;
 
 class ListerCell extends Cell
 {
-    protected array $items = [];
+    private array $items = [];
 
     public function getItemsProperty()
     {
```
I looked into the code, but I'm not sure how to fix.